pkgs/development/python2-modules/typing/default.nix

```diff
@@ -0,0 +1,32 @@
+{ lib, buildPythonPackage, fetchPypi, pythonOlder, isPy3k, isPyPy, unittestCheckHook
+, pythonAtLeast }:
+
+let
+  testDir = if isPy3k then "src" else "python2";
```

Comment:
Can we remove this?

Reply:
I think it would be easier to leave this merge request as a revert, though unsure about that change would need to look into it.